planning/mission_planner/src/lanelet2_plugins/default_planner.cpp

```diff
@@ -380,18 +406,22 @@ PlannerPlugin::LaneletRoute DefaultPlanner::plan(const RoutePoints & points)
     const auto local_route_sections = route_handler_.createMapSegments(path_lanelets);
     route_sections = combine_consecutive_route_sections(route_sections, local_route_sections);
   }
-
-  if (!is_goal_valid(points.back(), all_route_lanelets)) {
```

Comment:
Thanks for reviewing, at the corners like below image, the goal footprint exceeds lane and it is rejected. I did this way because avoid this situation,  but you're right, this can have dangerous consequences. I updated it. :+1: 

<p align="center">
<img src="https://user-images.githubusercontent.com/56237550/212865420-1817b158-e2fc-462f-9536-d359054eeaaa.png" width="40%"/>
</p>